integration-tests/bridges/README.md

```diff
@@ -0,0 +1,116 @@
+cp ./target/release/chain-spec-generator ~/local_bridge_testing/bin/chain-spec-generator-kusama
+cp ./target/release/chain-spec-generator ~/local_bridge_testing/bin/chain-spec-generator-polkadot
+
+# in another folder of your choice, build the integritee-collator with sudo
+cd
+git clone https://github.com/integritee-network/parachain.git
+cd parachain
+git apply ./integration-tests/bridges/sudo-integritee.patch
+cargo build --release
+cp ./target/release/integritee-collator ~/local_bridge_testing/bin/
+```
+
+## automated testing (manually triggered)
+
+After that, you can run `./run-tests.sh <test_name>` command in `runtimes/integration-tests/bridges`
+.
+E.g. `./run-test.sh 0001-polkadot-kusama-asset-transfer`.
+or
+E.g. `FRAMEWORK_REPO_PATH=/home/username/polkadot-sdk ./run-test.sh 0001-polkadot-kusama-asset-transfer`.
+
+## manual testing
+
+If you'd like to interact with the local test networks, run this instead
+
+```
+./run-test.sh 0000-manual
+``` 
+
+then you can point your browser to
+
+* Integritee Network (Kusama) https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9144#
+* Integritee Network (Polkadot) https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9244#
+* Asset Hub Kusama https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9010#
+* Asset Hub Polkadot https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9910#
+* Bridge Hub Kusama https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A8945#
+* Bridge Hub Polkadot https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A8943#
+
+the setup is quite brittle. You should run the sanity check before testing other things:
+
+```bash
+cd ../dry-run
+bun xcm-sanity-check.ts
+```
+
+If any check fails, you may want to re-run one of the init steps manually, e.g.
+`./helper.sh init-asset-hub-polkadot-local` or `./helper.sh init-asset-hub-kusama-local`. See interventions section
+below.
+
+### a few calls to try
+
+XCMv5
+
+* send 1 KSM from KAH to IK(
+  Alice): https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9010#/extrinsics/decode/0x1f0b050101007d1f0500010100d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0504010000070010a5d4e80000000000
+* send 1 DOT from PAH to IP(
+  Alice) https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9910#/extrinsics/decode/0x1f0b05010100dd1f0500010100d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d05040100000700e40b54020000000000
+
+### interventions
+
+this setup is brittle and it can happen that not all setup calls succeed. To iterate a setup step, do:
+
+```
+cd integration-tests/bridges/environments/polkadot-kusama
+export ENV_PATH=~/integritee/parachain/integration-tests/bridges/environments/polkadot-kusama
+export FRAMEWORK_PATH=~/local_bridge_testing/downloads/polkadot-sdk/bridges/testing/framework/
+source "$FRAMEWORK_PATH/utils/bridges.sh"
+source "$FRAMEWORK_PATH/utils/zombienet.sh"
```

Comment:
Hmm, at this line I get the following error in wsl:

```
/home/clang/local_bridge_testing/downloads/polkadot-sdk/bridges/testing/framework//utils/zombienet.sh:source:3: no such file or directory: /common.sh
```

I don't know the issue, the line throwing the error is:

```bash
#!/usr/bin/env bash

source "${BASH_SOURCE%/*}/common.sh"
```

And I checked that the file exists. It seems than in wsl/zsh the `BASH_SOURCE` is not set.

Reply:
Solution: just open a bash session within zsh.